gcp-data-drive/Dockerfile

```diff
@@ -19,14 +19,25 @@
 FROM golang:1.13 as builder
 
 # Create and change to the app directory.
+WORKDIR /app
 
+# Retrieve application dependencies.
+# This allows the container build to reuse cached dependencies.
+COPY go.* ./
+RUN go mod download
 
-FROM gcr.io/distroless/static
+# Copy local code to the container image.
+COPY .  ./
+
+WORKDIR /app/cmd/webserver
+# Build the binary.
+RUN GO111MODULE=on CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -mod=readonly -v -o gcp-data-drive
```

Comment:
What does -mod=readonly do?

Reply:
Added comments to the file to explain. The TLDR is - It forces the build to fail if there any an issue with the go.mod that can be auto resolved by auto editing the go.mod.